service/analysis.gradle

```diff
@@ -21,15 +21,15 @@ spotbugs {
 	excludeFilter = file('../spotbugsExclude.xml')
 }
 
-spotbugsMain {
-	reports {
-		sarif { enabled = true }
+tasks.spotbugsMain {
+	reports.create("sarif") {
+		required = true
 	}
 }
 
 spotbugsTest { enabled = false }
 
-task runMinnieKenny(type: Exec) {
+tasks.register('runMinnieKenny', Exec) {
```

Comment:
[perhaps in a separate ticket] We should remove anything related to `minnieKenny`, the secret checker is deployed by BITS to devs on their macbooks.

Reply:
Yes, and spotbugs should be removed too since we use sonar. I can make a ticket